frontend/src/handlers/createProjectSubmitHandler.tsx

```diff
@@ -0,0 +1,73 @@
+import apiPaths from "../properties/apiPaths";
+import Router from "next/router";
+import axios from "axios";
+import {
+    AxiosConf,
+    AxiosFormConfig,
+    getEntityFromFullUrl,
+    getIdFromUrl,
+    ManyToManyAxiosConf,
+} from "../api/calls/baseCalls";
+import { Project } from "../api/entities/ProjectEntity";
+import applicationPaths from "../properties/applicationPaths";
+import { ProjectSkill } from "../api/entities/ProjectSkillEntity";
+
+export interface ProjectCreationValues {
+    projectName: string;
+    projectInfo: string;
+    versionManagement: string;
+    partnerName: string;
+    partnerWebsite: string;
+    skills: string[];
+    skillInfos: string[];
+    coaches: string[];
+}
+
+export type ProjectCreationProps = {
+    submitHandler: (values: ProjectCreationValues) => void;
+};
+
+export async function createProjectSubmitHandler(values: ProjectCreationValues) {
+    const ownUser = await axios.get(apiPaths.ownUser, AxiosConf);
```

Comment:
Oh! I just read this. Maybe we should wait to merge this PR until #363. Is merged. When we have that you have a nice hook to get the current user. (I think It would be cleaner to work that way.)

Reply:
true, I was going to wait for that PR

Reply:
Oh, this is not resolved?